.gitignore

```diff
@@ -1,6 +1,6 @@
 composer.phar
 vendor/
-
+.idea*
```

Comment:
Theres a really good repo by github, https://github.com/github/gitignore/ that's got a lot of good gitignore information.

They list all of the IDE and system specific ones in the [Global](https://github.com/github/gitignore/tree/master/Global) directory.

Its a pretty good resource.